com.unity.render-pipelines.universal/Runtime/ScriptableRenderer.cs

```diff
@@ -555,6 +556,14 @@ public virtual void SetupLights(ScriptableRenderContext context, ref RenderingDa
         {
         }
 
+        /// <summary>
+        /// Override this method to change the render state or prepare for transparent objects when transparent passes were enqueued.
+        /// </summary>
+        /// <param name="cmd"></param>
+        public virtual void BeforeTransparent(CommandBuffer cmd, ref RenderingData renderingData)
+        {
+        }
```

Comment:
I'm reluctant to accept adding new public API, users can inject custom code before transparent with a custom pass, so I think this would be more ways of doing the same thing which leads to confusion.

Reply:
I removed it and restored them to the original codes